src/Microsoft.Azure.IIoT.Services.OpcUa.Onboarding/src/Program.cs

```diff
@@ -94,13 +94,14 @@ public class Program {
 
             var config = new Config(ServiceInfo.ID, configuration);
             var builder = new ContainerBuilder();
-
+            
             // Register configuration interfaces
             builder.RegisterInstance(config)
                 .AsImplementedInterfaces().SingleInstance();
 
             // register logger
-            builder.RegisterLogger(LogEx.Console());
+            var logger = new LoggerConfiguration().ApplicationInsights(configuration).CreateLogger();
```

Comment:
Add another method to LogEx, same as the Console() and just rename Console to the new method name.